packages/block-library/src/image/interactivity.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Internal dependencies
+ */
+import { store } from '../utils/interactivity';
+
+const raf = window.requestAnimationFrame;
+// Until useSignalEffects is fixed: https://github.com/preactjs/signals/issues/228
+const tick = () => new Promise( ( r ) => raf( () => raf( r ) ) );
+
+const focusableSelectors = [
+	'a[href]',
+	'area[href]',
+	'input:not([disabled]):not([type="hidden"]):not([aria-hidden])',
+	'select:not([disabled]):not([aria-hidden])',
+	'textarea:not([disabled]):not([aria-hidden])',
+	'button:not([disabled]):not([aria-hidden])',
+	'iframe',
+	'object',
+	'embed',
+	'[contenteditable]',
+	'[tabindex]:not([tabindex^="-"])',
+];
+
+store( {
+	actions: {
+		core: {
+			image: {
+				showLightbox: ( { context } ) => {
+					context.core.image.initialized = true;
+					context.core.image.lightboxEnabled = true;
+					context.core.image.lastFocusedElement =
+						window.document.activeElement;
+					context.core.image.scrollPosition = window.scrollY;
+
+					context.core.image.documentElement =
+						document.documentElement;
+					context.core.image.adminElement =
+						document.querySelector( '#wpadminbar' );
+					context.core.image.siteBlocksElement =
+						document.querySelector( '.wp-site-blocks' );
+
+					context.core.image.documentElement.classList.add(
+						'has-lightbox-open'
+					);
```

Comment:
What is the point of storing `document.documentElement` in the context? If we are just using it here, couldn't we do directly `document.documentElement.classList.add( 'has-lightbox-open')?